src/services/message.service.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseChatroomDto, BaseMessageDto, CreateMessageDto } from '@/dtos';
+import { Chatroom, Messages } from '@/entities/nosql';
+import { HttpException } from '@/exceptions';
+import { model, ObjectId } from 'mongoose';
+
+class MessageService {
+  public async getChatroomMessages(id: ObjectId): Promise<BaseMessageDto[]> {
+    const chatroom = model('Chatroom', Chatroom);
+    const findMessages: BaseChatroomDto = await chatroom.findOne({ id });
+    if (!findMessages.messages) throw new HttpException(404, 'No Messages');
```

Comment:
I don't think we really need this. If we get an empty list in the frontend, we just display nothing